### PR TITLE
Updated broken links 

### PR DESCRIPTION
**Pull request by sushreebarsa**
# Description

> :memo: Please include a summary of the change. 
>  
> * Please also include relevant motivation and context.  
> * List any dependencies that are required for this change.  

## Type of change

For a new feature or function, please create an issue first to discuss it
with us before submitting a pull request.

Note: Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Documentation update
- [ ] TensorFlow 2 migration
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] A new research paper code implementation
- [ ] Other (Specify)

## Tests

> :memo: Please describe the tests that you ran to verify your changes.
>  
> * Provide instructions so we can reproduce.  
> * Please also list any relevant details for your test configuration.  

**Test Configuration**:

## Checklist

- [ ] I have signed the [Contributor License Agreement](https://github.com/tensorflow/models/wiki/Contributor-License-Agreements).
- [ ] I have read [guidelines for pull request](https://github.com/tensorflow/models/wiki/Submitting-a-pull-request).
- [ ] My code follows the [coding guidelines](https://github.com/tensorflow/models/wiki/Coding-guidelines).
- [ ] I have performed a self [code review](https://github.com/tensorflow/models/wiki/Code-review) of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.

### CODE REVIEW

**Review by saberkun — APPROVED** [review]

File: official/README-TPU.md [hunk 1]
```diff
@@ -2,21 +2,21 @@
...
-    [BERT FineTuning with Cloud TPU](https://cloud.google.com/tpu/docs/tutorials/bert-2.x) provides step by step instructions on Cloud TPU training. You can look [Bert MNLI Tensorboard.dev metrics](https://tensorboard.dev/experiment/LijZ1IrERxKALQfr76gndA) for MNLI fine tuning task.
+    [BERT FineTuning with Cloud TPU](https://cloud.google.com/tpu/docs/tutorials/bert-2.x) provides step-by-step instructions on Cloud TPU training. You can look [Bert MNLI Tensorboard.dev metrics](https://tensorboard.dev/experiment/LijZ1IrERxKALQfr76gndA) for MNLI fine tuning task.
 *   [transformer](nlp/transformer): A transformer model to translate the WMT
     English to German dataset.
-        [Training transformer on Cloud TPU](https://cloud.google.com/tpu/docs/tutorials/transformer-2.x) for step by step instructions on Cloud TPU training.
+        [Training transformer on Cloud TPU](https://cloud.google.com/tpu/docs/tutorials/transformer-2.x) for step-by-step instructions on Cloud TPU training.
 
 ## Computer Vision
 
-*   [efficientnet](vision/image_classification): A family of convolutional
```
Comment (saberkun) [inline official/README-TPU.md:15]: Let's don't point to the keras application.
Could we use https://github.com/tensorflow/models/blob/master/official/vision/modeling/backbones/efficientnet.py?

**Review by saberkun — CHANGES_REQUESTED** [review]

File: official/README-TPU.md (hunk 1 above)
Comment (yeqingli) [reply official/README-TPU.md:15]: If document needed, can be this one: https://www.tensorflow.org/api_docs/python/tfm/vision/backbones/EfficientNet

Comment (sushreebarsa) [reply official/README-TPU.md:15]: I have made the changes as needed. Thank you!

**Comment by gbaned** [comment]
Hi @saberkun, Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @saberkun, Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @Chen-Zhe, Can you please review this PR ? Thank you!

**Review by yeqingli — APPROVED** [review]

**Comment by laxmareddyp** [comment]
Closing this PR as these changes submitted through [internal CL](https://github.com/tensorflow/models/commit/8e9f80a27e4abd2ed889701cd34d42ef2a6e9373)

**Pull request closed without merging**